resources/assets/js/components/JobDetails/JobDetails.tsx

```diff
@@ -950,7 +955,10 @@ const JobDetails: React.FunctionComponent<
                         province={intl.formatMessage(
                           provinceName(Number(values.province)),
                         )}
+                        education={values.education}
                         termLength={Number(values.termLength)}
+                        telework={String(values.telework)}
```

Comment:
I think it makes sense to pass the already localized versions of telework and flexhours here, rather than duplicating all the translations within the JobPreview component.

It also tightly couples the preview to the JobDetails form.

Reply:
Agreed. I'm using a different approach now. 👍